luigi_pipeline/seqr_mito_loading.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import sys
+import pkg_resources
+import pprint
+
+import luigi
+import hail as hl
+
+from lib.model.mito_mt_schema import SeqrMitoVariantsAndGenotypesSchema, SeqrMitoVariantSchema, SeqrMitoGenotypesSchema
+from luigi_pipeline.seqr_loading_optimized import BaseVCFToGenotypesMTTask, BaseMTToESOptimizedTask
+from seqr_loading import SeqrVCFToMTTask
+
+logger = logging.getLogger(__name__)
+
+
+class SeqrMitoVariantMTTask(SeqrVCFToMTTask):
```

Comment:
I don't think we need this task at all and its kind of clunky to read/ maintain. Lets remove it